aiida/orm/computers.py

```diff
@@ -264,6 +264,20 @@ def _default_mpiprocs_per_machine_validator(cls, def_cpus_per_machine: Optional[
                 'do not want to provide a default value.'
             )
 
+    @classmethod
+    def _default_memory_per_machine_validator(cls, def_memory_per_machine: Optional[int]) -> None:
```

Comment:
Don't think you are actually using this. Did you forget to add this to `Computer.validate`?

Reply:
This comment still stands correct? This is not being called in the current code.

Reply:
I think it is correct. I now call it in `set_default_memory_per_machine`. I also renamed it to `default_memory_per_machine_validator` as, presumably, it will be called outside of the class. In case we want to do the validation elsewhere.